e2e-tests/playwright/e2e/authProviders/basic-authentication.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { test, Page, expect } from '@playwright/test';
+import { Common, setupBrowser } from '../../utils/Common';
+import { UIhelper } from '../../utils/UIhelper';
+import * as constants from '../../utils/authenticationProviders/constants';
+import { logger } from '../../utils/authenticationProviders/Logger';
```

Comment:
It is writing to a file. It was requested by the rest of QE team since the configurations will be changed and destroyed during runtime; it's a way to keep event or config dumps from instances that do not exists anymore once the test finishes. This could probably be a common approach to all tests other than just printing to console.

Reply:
If this is approach that other tests cases should adopt wouldn't ti be better to make it more generic so it can be re-used by other scenarios?